src/components/common/GetWinnerButton.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+// import PropTypes from 'prop-types'
+
+const GetWinnerButton = props => {
+  const handleSubmit = e => {
+    e.preventDefault();
+    axios
+      .get('http://localhost:5000/api/game/results?squadId=1')
```

Comment:
I'm assuming the squad ID is hardcoded for now, just making note of it. Also we should probably be using the apiAuthGet() function to do the axios call to ensure those making the request are authenticated.